pkg/instrumentation/sdk.go

```diff
@@ -186,6 +186,13 @@ func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alph
 		}
 	}
 
+	// Move OTEL_RESOURCE_ATTRIBUTES to last position.
+	// All used env vars as value have to be configured before
+	// to avoid incorrect attributes values.
+	idx = getIndexOfEnv(container.Env, constants.EnvOTELResourceAttrs)
```

Comment:
@pavolloffay I think it's not possible in case when `OTEL_RESOURCE_ATTRIBUTES` is already configured - in this case operator is just modifying value of the variable. That's why at the end we have to change order and set it as last one.

Reply:
that is right, or the operator could be adding env vars always to the front of the env var list which is more expensive.